neural_modelling/src/neuron/c_main.c

```diff
@@ -83,11 +83,15 @@ struct neuron_provenance {
     uint32_t spike_processing_get_max_filled_input_buffer_size;
     //! the number of times the TDMA fully missed its slots
     uint32_t n_tdma_mises;
+    //! Maximum backgrounds queued
+    uint32_t max_backgrounds_queued;
+    //! Background queue overloads
+    uint32_t n_background_queue_overloads;
 };
 
 //! values for the priority for each callback
 typedef enum callback_priorities {
-    MC = -1, DMA = 0, USER = 0, SDP = 1, TIMER = 2
+    MC = -1, DMA = 0, USER = 0, SDP = 1, TIMER = 0, BACKGROUND = 1
```

Comment:
Is there a reason why the order is different here compared to the delay extension code?  (TIMER is before USER in the enum there...)

Reply:
The order isn't critical but I have changed it to be more consistent anyway